Makefile

```diff
@@ -62,19 +62,27 @@ work/imagenet/parsed_bb.csv: work/.imagenet_training_bbox_sentinel \
 	python motion_tracker/data_setup/parse_imagenet_bb.py data/imagenet/bb/ \
 		work/imagenet/parsed_bb.csv
 
+# Run after images are downloaded, to filter out those images 
```

Comment:
Normally with make, I don't expect to keep track of when to run things. Anything we need this for will depend on it.  

And anything that has to be run before this can be run automatically when we try to build this.  

Is this a special case?